templates/20years/index.html

```diff
@@ -0,0 +1,1402 @@
+          </p>
+          <div class="p-section--shallow">
+            {{ image (
+            url="https://assets.ubuntu.com/v1/eb725025-post 6.png",
+            alt="",
+            width="1512",
+            height="852",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+            }}
+          </div>
+          <hr class="p-rule--muted" />
+          <div class="p-heading-icon--small">
+            <div class="p-heading-icon__header">
+              <img class="p-heading-icon__img"
+                   src="https://assets.ubuntu.com/v1/0da07e88-X%20Logo%20-%20Updated%202.png"
+                   alt="X logo" />
+              <p class="p-heading-icon__title">
+                <a href="https://x.com/shaakunthala/status/1819043688753889436?ref_src=twsrc%5Etfw">See post&nbsp;&rsaquo;</a>
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="p-card--social-media">
+          <div class="p-heading-icon">
+            <div class="p-heading-icon__header">
+              <img class="p-heading-icon__img"
+                   src="https://assets.ubuntu.com/v1/62a77319-@pleia2.png"
+                   alt="" />
+              <p class="p-heading-icon__title">
+                <strong><small>Tweet by
+                @pleia2</small></strong>
+              </p>
+            </div>
+          </div>
+          <p>
+            I haven't been on the <a href="https://x.com/hashtag/Ubuntu?src=hashtag_click">#Ubuntu</a> wagon the whole time, but I did do my first install on March 12th, 2005 (I know the exact date because I also signed up to the Ubuntu Forums that day), and requested a copy of the second release!
+          </p>
+          <p>
+            Happy birthday 15th birthday! <a href="https://x.com/hashtag/Ubuntu15?src=hashtag_click">#Ubuntu15</a> 😁
+          </p>
+          <div class="p-section--shallow">
+            {{ image (
+            url="https://assets.ubuntu.com/v1/a5a694bc-post 8.png",
+            alt="",
+            width="1512",
+            height="852",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+            }}
+          </div>
+          <hr class="p-rule--muted" />
+          <div class="p-heading-icon--small">
+            <div class="p-heading-icon__header">
+              <img class="p-heading-icon__img"
+                   src="https://assets.ubuntu.com/v1/0da07e88-X%20Logo%20-%20Updated%202.png"
+                   alt="" />
+              <p class="p-heading-icon__title">
+                <a href="https://x.com/pleia2/status/1186143153276895233?ref_src=twsrc%5Etfw">See post&nbsp;&rsaquo;</a>
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="col-3 col-medium-2">
```

Comment:
I think this might be a mistake in the copydoc. I have left a comment here: https://docs.google.com/document/d/1vlENASSxNJdnK5tRuNYEZGVbh_qrVN0lhon8SOYo6A0/edit?disco=AAABQBWBniQ